src/modules/collections/controllers/collections.controller.ts

```diff
@@ -290,6 +291,33 @@ export class CollectionsController {
 			referer
 		);
 
+		// Get all the objects from the original collection and add them to the new collection
+		let folderObjects: IPagination<Partial<IeObject>>;
+		try {
+			folderObjects = await this.collectionsService.findObjectsByCollectionId(
+				collectionId,
+				collection?.userProfileId,
+				{ size: 1000 },
+				referer
+			);
+			// TODO: make it possible to insert all objects to the new collection at once
+			await Promise.all(
+				folderObjects?.items.map(async (item) => {
+					await this.collectionsService.addObjectToCollection(
+						createdCollection.id,
+						item?.schemaIdentifier,
+						referer
+					);
+				})
+			);
+		} catch (err) {
+			if (err.name !== 'NotFoundException') {
```

Comment:
also add a console.error with the actual error, so we can debug problems on the server by looking at the logs